src/app/organizations/organization-details/organization-details.component.scss

```diff
@@ -90,35 +82,37 @@
 
 .manage__members {
   .col-lg-7,
-  .col-lg-5,
-  .col-lg-2 {
+  .col-lg-6,
+  .col-lg-1,
+  .col-xl-5,
+  .col-xl-2 {
     padding: 0;
   }
 
   .invite {
     &-members {
       padding: 15px 0;
-      margin: 0 -20px;
       background-color: $background-color;
     }
     &__form {
-      margin: 0 8px;
+      margin: 0 15px;
       display: flex;
       justify-content: center;
       padding-bottom: 8px;
       .form__input {
-        margin-left: -18px;
+        width: calc(85% + 18px);
```

Comment:
can you tell me roughly why these numbers? 18px? just asking.

Reply:
These numbers can be changed. When we add the ability for users to configure their app (add their colors and change border-radius through the app) we are going to need some negative margin so that everything looks decent. At the moment of writing this code, I thought that 18px should be enough to cover most use cases of user increasing border-radius.